### PR TITLE
DOC-4502

### PR DESCRIPTION
**Pull request by jamesnocentini**

### CODE REVIEW

**Review by rajagp — COMMENTED** [review]
The changes by themselves are fine. I would suggest describing the use case. Something like this (Please edit as needed)
"Shared Bucket access will enable document changes to Couchbase server bucket made either  through the Couchbase SDKs or directly through the N1QL queries to be synchronized to the Couchbase Lite enabled clients or vice versa" ...

**Comment by jamesnocentini** [comment]
@rajagp is that better? (pls approve the PR if there's nothing to change)

File: modules/ROOT/pages/shared-bucket-access.adoc [hunk 1]
```diff
@@ -1,18 +1,17 @@
 = Mobile-Web Data Sync
 :url-downloads: https://www.couchbase.com/downloads
 
-Starting in Sync Gateway 1.5, you can seamlessly extend an existing Couchbase Server deployment to connect with remote edge devices that are occasionally disconnected or connected.
+Shared Bucket Access enables document changes in a Couchbase Server bucket to be made through Couchbase SDKs, N1QL queries or Sync Gateway and vice versa.
+The following diagram shows different clients (mobile, web, desktop) writing data to the same bucket in Couchbase Server.
+This feature is available since Sync Gateway 1.5 and Couchbase Server 5.0.
+
```
Comment (rajagp) [inline modules/ROOT/pages/shared-bucket-access.adoc:7]: you are missing the key word "synchronized".  Shared bucket access allows server side and client side changes to be automatically synchronized

Comment (jamesnocentini) [reply modules/ROOT/pages/shared-bucket-access.adoc:7]: Updated. I had missed the importance of that aspect.
Dropped the keyword "enabled" in the context of Couchbase Lite.

**Pull request merged**